perf/roles/trex-ndr/tasks/main.yml

```diff
@@ -3,14 +3,14 @@
 - name: upload traffic profile
   template:
     src: '{{profile}}_profile.py'
-    dest: '{{trex_dir}}/scripts/{{profile}}_profile.py'
+    dest: '{{ trex_install_dir }}/{{trex_version}}/{{profile}}_profile.py'
```

Comment:
I don't like having the version in the install path. We don't plan on installing multiple versions. This should be installed flat in `{{trex_install_dir}}`